index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title id="title">Microverse Trials</title>
```

Comment:
The id="title" doesn't need to be here since the <head> doesn't show up on your page and doesn't need to be styled.

Reply:
Addressed 🔥✔